app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="recipes-show-container">
+  <h2><%= @recipe.name %></h2>
+
+  <div class="preparation-and-public">
+    <p>Preparation Time: <%= @recipe.preparation_time %> minutes</p>
+    <p>Public: <%= @recipe.public ? 'Yes' : 'No' %></p>
+    <%= button_to 'Toggle Public', url_for(action: 'toggle_public', controller: 'recipes', id: @recipe.id), remote: true, method: :patch %>
+  </div>
+
+  <p>Cooking Time: <%= @recipe.cooking_time %> minutes</p>
+  <p>Description: <%= @recipe.description %></p>
+  
+  <div class="generate-and-add">
+    <%= link_to 'Generate Shopping List', shopping_list_path, class:"button-white-black" %>
+    <%= link_to 'Add Ingredient', new_recipe_food_path(recipe_id: @recipe.id), class:"button-white-black" %>
+  </div>
+  <hr>
+
+  <% if @recipe.user == current_user %>
+    <%= form_with(model: @recipe, url: recipe_path(@recipe), method: :delete, data: { confirm: 'Are you sure you want to delete this recipe?' }) do |form| %>
+    <%= form.submit 'Delete', class:"button-white-black" %>
+  <% end %>
+  <% end %>
+
+  <h3>Ingredients:</h3>
+  <ul class="ingredients-container">
+    <% @recipe.recipe_foods.each do |recipe_food| %>
+      <li class="ingredients">
+        <span><%= recipe_food.food.name %></span>
+        <span>(<%= recipe_food.quantity %> units)</span>
+        <span><%= link_to 'Update', edit_quantity_recipe_food_path(recipe_food), class:"button-white-black" %></span>
+        <span>
+          <%= form_with(model: recipe_food, url: recipe_food_path(recipe_food), method: :delete, data: { confirm: 'Are you sure you want to remove this ingredient?' }) do |form| %>
+            <%= form.submit 'Remove', class:"button-white-black" %>
+          <% end %>        
+        </span>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
[REQUIRED] Kindly ensure that this is in. table with headings just like [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_food.png?token=GHSAT0AAAAAACHZNTDRFYRG7Z66ZJPMFFXUZJHHW5Q). I cant see the headings